test/kind/kubernetes-1.17.2.yaml

```diff
@@ -0,0 +1,5 @@
+kind: Cluster
+apiVersion: kind.sigs.k8s.io/v1alpha3
+nodes:
+- role: control-plane
+  image: kindest/node:v1.17.2
```

Comment:
Do we need to override the image, instead of using the latest version that comes with our version of kind?

Reply:
the idea was having multiple configurations in `test/kind`, so we can have CI checks for a number of kubernetes versions, but as for now I can remove it

Reply:
as for now removed this, and using KIND 0.8.1